### PR TITLE
monitor: add tiflash to grafana dashboard

### PR DESCRIPTION
**Pull request by marsishandsome**
### What problem does this PR solve?

Problem Summary: add tiflash to grafana dashboard

### What is changed and how it works?

image

### Check List 

Tests 

- No code

### CODE REVIEW

File: metrics/grafana/overview.json [hunk 1]
```diff
@@ -235,6 +235,13 @@
               "intervalFactor": 2,
               "legendFormat": "Kafka_exporter",
               "refId": "L"
+            },
+            {
+              "expr": "\ncount(probe_success{group=\"tiflash\"} == 1)",
```
Comment (lzmhhh123) [inline metrics/grafana/overview.json:240]: Why not move it below the `tikv`?

File: metrics/grafana/overview.json [hunk 2]
```diff
@@ -406,6 +413,13 @@
               "intervalFactor": 2,
               "legendFormat": "Kafka_exporter",
               "refId": "L"
+            },
+            {
+              "expr": "\ncount(probe_success{group=\"tiflash\"} == 0)",
```
Comment (lzmhhh123) [inline metrics/grafana/overview.json:418]: ditto.

File: metrics/grafana/overview.json (hunk 1 above)
Comment (marsishandsome) [reply metrics/grafana/overview.json:240]: so your suggestion is to move `tiflash` below `tikv`?

Comment (lzmhhh123) [reply metrics/grafana/overview.json:240]: Yes.

Comment (marsishandsome) [reply metrics/grafana/overview.json]: fixed

**Comment by marsishandsome** [comment]
@lzmhhh123 PTAL again

**Pull request merged**